### PR TITLE
Refactoring For loop

### PR DESCRIPTION
**Pull request by miguelsolano**
A refactoring in order to remove a for loop and possibly optimize this a bit.

### CODE REVIEW

File: lib/commands.js [hunk 1]
```diff
@@ -58,10 +58,9 @@ var commands = module.exports.commands = [
...
-  for (var i=0, len=commands.length; i<len; i++) {
-    if (commands[i].toLowerCase() === command) {
-        return true
-    }
-  }
+    command = command.toLowerCase();
+    var lowerCaseCommands = commands.map(function (item) {
+        return item.toLowerCase();
+    });
+    return lowerCaseCommands.indexOf(command) !== -1 ? true : false;
```
Comment (fentas) [inline lib/commands.js]: The ternary operator is unnecessary here.

**Comment by miguelsolano** [comment]
@fentas Added edits to the PR

**Comment by justinvforvendetta** [comment]
that does look much nicer. sorry i just saw this!

**Comment by miguelsolano** [comment]
@justinvforvendetta No worries! I know you're swamped! I'd love to become a collaborator to Verge in any way that I could. Let me know if you all need help with anything always love developing PHP and JS. I thought about making this into an npm module that your could download, but I wanted to wait to clear with you all first 👍

**Pull request merged**